.github/workflows/build-docs-generation-docker-image.yml

```diff
@@ -0,0 +1,26 @@
+name: Build Docker image used for docs generation
```

Comment:
Can we shorten the title just a smidge? Maybe "Build Docker image (docs)"

Reply:
Done. Also renamed the workflow file itself to reflect that title in the filename.